src/components/Dashboard/Balance.js

```diff
@@ -218,7 +218,9 @@ const LatestActivity = ({ activity, distribution }) => {
             color: ${theme.content};
           `}
         >
-          {convertToString(activity.type, activity.direction)}
+          {activity.isImmediate && !activity.isEffective
+            ? 'Effective next term'
```

Comment:
Wondering if we should have different text for the `Inactive` part of the balance module to clarify where the inactive portion has moved (to be withdrawal or activated).

Reply:
hmm yeah perhaps doesn't make sense to show the `effective next term` message there since is not something that is relevant for that balance and we can just return the `Activated`.